index.html

```diff
@@ -1,80 +1,96 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <title>SWE101</title>
-    <style>
-      * {
-        box-sizing: border-box;
-      }
 
-      body {
-        font-family: sans-serif;
-        margin-left: 30px;
-        margin-right: 30px;
-      }
+<head>
```

Comment:
Sorry about this- you're getting index.html changes because we hadn't linted the index.html file in the base repo previously. Have fixed so future batches shouldn't see these changes in index.html 🙏